components/Alert/index.jsx

```diff
@@ -0,0 +1,27 @@
+import Image from 'next/image';
+
+export const alertTestIds = {
+  icon: 'alert-icon',
+  title: 'alert-title',
+  description: 'alert-description',
+};
+
+const Alert = ({ icon, title, description }) => {
+  return (
+    <div className="w-full bg-notification backdrop-blur-2.5xl flex p-4 rounded-xl">
+      <span data-test-id={alertTestIds.icon}>
+        <Image src={icon} alt="alert" />
```

Comment:
Should we also set the width and height of the icon to `30px`?
![Screen Shot 2565-03-31 at 11 29 50](https://user-images.githubusercontent.com/1772999/160976841-2d575217-08ca-4ecb-847d-db29cec75ff0.png)

Reply:
done in 98bcf8c4034b7b2509d56f4680084e3416499844